ci/integration/logging/run_sg.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/env bash
+# CI script for UBI8 job
+# purpose: spawn sg-core to process messages sent by rsyslog
+
+set -ex
+
+# enable required repo(s)
+cat > /etc/yum.repos.d/fedora-eln.repo <<EOF
```

Comment:
Huh I learned a new trick today that if you surround the EOF in single quotes like `'EOF'` it won't expand the `$` etc.

```
cat > /tmp/foo <<'EOF'
> $(bar) foo
> EOF

cat /tmp/foo 
$(bar) foo
```